modules/next/src/Plugin/Next/Revalidator/CacheTag.php

```diff
@@ -56,29 +56,20 @@ public function revalidate(EntityActionEvent $event): bool {
       return FALSE;
     }
 
-    $tags = $entity->getCacheTags() ?? [];
-    if ($entity->getEntityTypeId() == 'menu_link_content') {
-      $tags[] = $entity->getEntityTypeId() . ':' . $entity->getMenuName();
-    }
-    else {
-      $tags[] = $entity->getEntityTypeId() . '_list';
-      $tags[] = $entity->getEntityTypeId() . '_list:' . $entity->bundle();
-    }
+    $cache_tags = implode(',', $entity->getCacheTags());
```

Comment:
@minnur I think the entity list tags we still have to add manually, at least in my test for nodes for example $entity->getCacheTags only gave `node:8`

Reply:
I didn't know about this, but apparently if we do this: 

```php
$entity->getEntityType()->getListCacheTags();
```

We should get the correct list tag. The one with the bundle at the end we still need to create manually.

<img width="882" alt="image" src="https://github.com/user-attachments/assets/746a68cf-537d-4f8c-a0b2-ffa0576317a2" />

Reply:
Good find! I added that to the plugin. See db2690b16b46033837c050a322dcc5ea7a3cf9f5
I noticed  `$entity->getEntityType()->getListCacheTags();` does not have entity_type_list:bundle. So I looked at `\Drupal\Entity\EntityBase` class and found a method `getListCacheTagsToInvalidate()` that adds that to the invalidation list and implemented the same logic in the plugin.